File: tests/dynamic_fixtures/go/ssrf_unsupported.go
// SSRF — unsupported fixture.
// Entry is a method on a struct; test sets confidence = Low.
// Expected verdict: Unsupported

package entry

import (
	"io"
	"net/http"
)

type HTTPClient struct{}

func (c *HTTPClient) Fetch(targetURL string) {
	resp, err := http.Get(targetURL)
	if err == nil {
		defer resp.Body.Close()
		io.Copy(io.Discard, resp.Body)
	}
}

File: tests/dynamic_fixtures/java/ssrf_unsupported.java
// SSRF — unsupported fixture.
// Entry is an instance method; test sets confidence = Low.
// Expected verdict: Unsupported

import java.io.*;
import java.net.*;

public class Entry {
    public void fetch(String url) throws Exception {
        new URL(url).openStream().close();
    }
}
